GeorgianMemenator/Assets.xcassets/Icons/dog.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "dog.png",
```

Comment:
You have added `dog.png`, ideally we want to have `.svg` files. Let it be for now, but consider in the future. 

There is an option on flaticon to download .svg version of the icon.

Reply:
<img width="334" alt="Screenshot 2023-01-10 at 22 25 52" src="https://user-images.githubusercontent.com/111251425/211632449-1010d200-22e3-4352-a49d-47e144b46193.png">

There was just one option and what's why I downloaded dog.png file

Reply:
You should open its details, but its okay no worries